processor/integrations/integrations.go

```diff
@@ -52,7 +52,7 @@ type PostParametersT struct {
 	Type          string                 `json:"type"`
 	URL           string                 `json:"endpoint"`
 	RequestMethod string                 `json:"method"`
-	UserID        string                 `json:"userId,,optional"`
+	UserID        string                 `json:"userId,omitempty"`
```

Comment:
And for future reference, can you please add the following comment explaining why its like that
`//optional is a custom tag introduced by us and is handled by GetMandatoryJSONFieldNames. Its intentionally added after two commas because the tag that comes after the first comma should be known by json parser`

Reply:
awesome, got it!